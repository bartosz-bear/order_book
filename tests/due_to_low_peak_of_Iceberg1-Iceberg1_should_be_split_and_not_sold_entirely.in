{"type": "Iceberg", "order": {"direction": "Buy", "id": 1, "price": 15, "quantity": 20, "peak": 10}}
{"type": "Limit", "order": {"direction": "Buy", "id": 2, "price": 15, "quantity": 20}}
{"type": "Limit", "order": {"direction": "Sell", "id": 3, "price": 15, "quantity": 30}}