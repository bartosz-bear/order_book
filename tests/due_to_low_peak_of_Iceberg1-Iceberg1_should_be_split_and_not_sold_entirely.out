{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}], "sellOrders": []}
{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}, {"id": 2, "price": 15, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 1, "price": 15, "quantity": 10}], "sellOrders": []}
{"sellOrderId": 3, "buyOrderId": 1, "price": 15, "quantity": 10}
{"sellOrderId": 3, "buyOrderId": 2, "price": 15, "quantity": 20}